test-network-function/lifecycle/suite.go

```diff
@@ -176,45 +156,26 @@ func testScaling(env *config.TestEnvironment) {
 	ginkgo.It(testID, func() {
 		ginkgo.By("Testing deployment scaling")
 		defer results.RecordResult(identifiers.TestScalingIdentifier)
+		defer restoreDeployments(env)
 
-		namespaceDeploymentsBackup := make(map[string]dp.DeploymentMap)
-		defer restoreDeployments(env, &namespaceDeploymentsBackup)
-
-		// Map to register the deployments that have been already tested
-		deploymentNames := make(map[string]bool)
-
-		for _, podUnderTest := range env.PodsUnderTest {
-			podName := podUnderTest.Name
-			namespace := podUnderTest.Namespace
-
-			// Get deployment name and check whether it was already tested.
-			// ToDo: Proper way (helper/handler) to do this.
-			podNameParts := strings.Split(podName, "-")
-			deploymentName := podNameParts[0]
-			msg := fmt.Sprintf("Testing deployment=%s, namespace=%s pod name=%s", deploymentName, namespace, podName)
-			log.Info(msg)
-			if _, alreadyTested := deploymentNames[deploymentName]; alreadyTested {
-				continue
-			}
+		if len(env.DeploymentsUnderTest) == 0 {
+			ginkgo.Fail("No test deployments found.")
```

Comment:
Move this test in the before each section, similar to pods/containers

Reply:
Do we really want to fail every test?

Reply:
as discussed, skip instead of fail

Reply:
Done.